src/third-party/country-state-city/index.js

```diff
@@ -0,0 +1,93 @@
+import country_json_1 from './lib/country.json';
+import state_json_1 from './lib/state.json';
+import city_json_1 from './lib/city.json';
```

Comment:
any reason why these variables names are like these? how about just `countries`, `states` and `cities`?

Reply:
@aviskarkc10 this was taken as is from country-stats-city npm module

Reply:
I got that, but maybe we can use better variable names?

Reply:
will it leave it this way for consistency with the same third-party module in patients app